xml/schema/Member/MembershipType.xml

```diff
@@ -115,7 +115,8 @@
   <field>
     <name>minimum_fee</name>
     <title>membership Type Minimum Fee</title>
-    <type>decimal</type>
+    <type>varchar</type>
+    <length>512</length>
```

Comment:
MembershipType.minimum_fee gets saved in civicrm_price_field_value.amount and retrieved from there.  That field is a varchar(512).  Which is why I used varchar(512), but anything that allows more decimals is fine by me :-)